topo/node/nokia/nokia.go

```diff
@@ -32,6 +33,9 @@ const (
 	// configuration reset is therefore done by reverting to this checkpoint
 	configResetCmd = "/tools system configuration checkpoint initial revert"
 	pushCfgFile    = "/home/admin/kne-push-config"
+
+	defaultCPU = "2"
```

Comment:
so at this point ...

defaultConstraints = node.Constraints{
   CPU: 2.0,
   Memory: 4 * 1 << 30, // gigabyte 
}

Reply:
then it is much clearer everwhere else